database/index.js

```diff
@@ -29,7 +29,7 @@ let locationSchema = mongoose.Schema({
 let Trip = mongoose.model('Trip', tripSchema);
 let Location = mongoose.model('Location', locationSchema);
```

Comment:
I'm surprised you are using mongoose with Postgres, how so?

Reply:
I'm not! This is inherited code and it will be pulled out.